channel/api/snippets/WorkspaceProvisioning/Program.cs

```diff
@@ -0,0 +1,174 @@
+            String sampleOffer = "Google Workspace Business Standard";
+            String samplePlan = "Commitment";
+
+            foreach (Offer offer in listOffersResponse.ToList())
+            {
+                String offerName = offer.Sku.MarketingInfo.DisplayName;
+                String offerPlan = offer.Plan.PaymentPlan.ToString();
+                if (offerName == sampleOffer && offerPlan == samplePlan)
+                {
+                    selectedOffer = offer;
+                    break;
+                }
+            }
+
+            CheckCloudIdentityAccountsExistRequest checkCloudIdentityAccountsExistRequest =
+                new CheckCloudIdentityAccountsExistRequest
+                {
+                    Parent = accountParent,
+                    Domain = customerDomain
+                };
+
+            CheckCloudIdentityAccountsExistResponse checkIdentityResponse =
+                client.CheckCloudIdentityAccountsExist(checkCloudIdentityAccountsExistRequest);
+
+            if (checkIdentityResponse.CloudIdentityAccounts.Count > 0)
+            {
+                throw new Exception(
+                    "Cloud identity already exists. Customer must be transferred. Out of scope for this codelab");
+            }
+
+            PostalAddress postalAddress = new PostalAddress
+            {
+                AddressLines = {
+                    "123 Main St"
+                },
+                PostalCode = "97224", RegionCode = "US"
+            };
+
+            Customer customer = new Customer
+            {
+                OrgDisplayName = "Acme Corp", OrgPostalAddress = postalAddress, Domain = customerDomain
+            };
+
+            CreateCustomerRequest createCustomerRequest =
+                new CreateCustomerRequest {
+                    Parent = accountParent, 
+                    Customer = customer
+                };
+
+            Customer createCustomerResponse = client.CreateCustomer(createCustomerRequest);
+
+            CloudIdentityInfo cloudIdentityInfo =
+                new CloudIdentityInfo
+                {
+                    AlternateEmail = "marty.mcfly@gmail.com",
+                    LanguageCode = "en-US"
+                };
+
+            AdminUser adminUser = new AdminUser
+            {
+                GivenName = "Marty", FamilyName = "McFly",
+                Email = "admin@" + createCustomerResponse.Domain
+            };
+
+            ProvisionCloudIdentityRequest provisionCloudIdentityRequest =
+                new ProvisionCloudIdentityRequest
+                {
+                    Customer = createCustomerResponse.Name,
+                    CloudIdentityInfo = cloudIdentityInfo,
+                    User = adminUser
+                };
+
+            var provisionCloudIdentityOp = client.ProvisionCloudIdentity(provisionCloudIdentityRequest);
+
+            PollSettings pollSettings = new PollSettings(Expiration.FromTimeout(TimeSpan.FromSeconds(30)),
```

Comment:
Did you confirm this was required to do polling? I'd rather be consistent across languages in the codelab

Reply:
I can confirm this is not needed.